src/browser-common/replicants.ts

```diff
@@ -10,10 +10,12 @@ export async function loadReplicants() {
 			}),
 			selectedPort: await replicant<string | null>("selectedPort", "glimpse-graphics.sync-settings", {defaultValue: null}),
 			selectedSport: await replicant<string>("selectedSport", "glimpse-graphics.sync-settings", {defaultValue: 'Hockey/Lacrosse'}),
-			status: await replicant<{ connected: boolean, bitrate: number }>("status", "glimpse-graphics.sync-settings", {
+			status: await replicant<{ connected: boolean, bitrate: number, error: boolean, errorMsg: string }>("status", "glimpse-graphics.sync-settings", {
 				defaultValue: {
 					connected: false,
-					bitrate: 0
+					bitrate: 0,
+					error: false,
```

Comment:
Could this be simplified to just the single `errorMsg` variable? If `errorMsg` is set then it is implied that `error` is also true.

Reply:
That is a good point. I believe that should cause no problems.